ansible/inventory/env/group_vars/all.yml

```diff
@@ -170,4 +170,15 @@ processing_zookeeper_ip: "{{ groups['processing-cluster-zookeepers'][0] }}"
 ### docker image pull secrets name and this will be created in kubernetes
 imagepullsecrets: "{{env}}-registry-secret"
 dialcode_host: "http://{{private_ingressgateway_ip}}/dial"
-dialcode_endpoint: "dialcode/v3/read/"
\ No newline at end of file
+dialcode_endpoint: "dialcode/v3/read/"
+
+# s3 storage config
+s3_storage_key: ""
+s3_storage_secret: ""
```

Comment:
The `s3_storage_secret` has to go into ansible secrets as the secret will be encrypted and stored in the ansible vault.

Reply:
set to private variables that will be picked up from the private repo